modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/SubscriptionsFilterByStatus/index.js

```diff
@@ -0,0 +1,50 @@
+import React, {useEffect, useMemo} from 'react';
+
+const SubscriptionsFilterByStatus = ({setSelectedStatus}) => {
+	const options = useMemo(
+		() => [
+			{
+				label: 'All',
+				value: 'All',
+			},
+			{
+				label: 'Active',
+				value: 'Active',
+			},
+			{
+				label: 'Expired',
+				value: 'Expired',
+			},
+			{
+				label: 'Future',
+				value: 'Future',
+			},
+		],
+		[]
+	);
+
+	const handleFilterChange = (event) => {
+		setSelectedStatus(event.target.value);
+	};
+
+	useEffect(() => {
+		setSelectedStatus(options[0].value);
+	}, [options, setSelectedStatus]);
```

Comment:
```suggestion
	useEffect(() => {
		setSelectedStatus(options[0].value);
	}, []);
```

Reply:
This component is in initial development in this branch. We implemented it only for testing purposes.